tests/integration/runners/test_fileserver.py

```diff
@@ -161,6 +163,10 @@ def test_file_list(self):
         self.assertIsInstance(ret['return'], list)
         self.assertTrue('grail/scene33' in ret['return'])
 
+    # Git doesn't handle symlinks in Windows. See the thread below:
+    # http://stackoverflow.com/questions/5917249/git-symlinks-in-windows
+    @skipIf(salt.utils.is_windows(),
+            'Git doesn\'t handle symlinks properly on Windows')
```

Comment:
I don't think this needs to be documented since I think @cachedout and I both figured that you were referring to gitfs, but if it should be documented anywhere, it would be in the fileserver documentation. However, I think we should take some care to make sure that the fileserver works as expected on Windows, and that means making sure that symbolic links in the fileserver backends are handled properly.